### ISSUE TITLE
Publish desktop version to Snapcraft

### ISSUE THREAD

**Issue by jgadsden** [OWASP/threat-dragon#241]
**Describe what problem your feature request solves**
It would be great to have Threat Dragon published on Snapcraft

**Describe the solution you'd like**
Something like the one for OWASP Amass https://snapcraft.io/amass
Maybe using electron builder: https://www.electron.build/configuration/snap.html
It would be good if this was automated from the release process

**Additional context**
Follow this blog if it helps:
https://abskmj.github.io/notes/posts/github/actions/setup-snapcraft/

Amass snapcraft yaml:
https://github.com/OWASP/Amass/blob/30f786aae86e44751f3ebbe2cebb9b25638c1934/snapcraft.yaml

**Comment by lreading** [comment]
It appears the name "threat-dragon" has been reserved on snapcraft.  Any idea who that was? (hopefully you?)  If not, there is a process for filing a dispute, but I'm hopeful whoever has it registered is still around. :)

**Comment by jgadsden** [comment]
yes, I am that guilty party. This is the snap-store username and also the registered name. I did it a while ago and did not fill out the info - this is now corrected.

We are going to have the same problem as Docker Hub - it has been registered as a single user which is not going to be convenient. If we can hop on to the docker owasp organisation then that will sort that one out ... maybe there is something similar for Cononical's snapcraft

I have renamed my `threat-dragon` as `threatdragon` - this may allow you in @lreading ?
